packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -418,6 +418,11 @@ - (NSView *)nextViewToFocusOutsideZone:(FocusZoneAction)action
 		nextViewToFocus = [self nextValidKeyView];
 		while([nextViewToFocus isDescendantOf:focusZoneAncestor])
 		{
+			if ([nextViewToFocus isEqual:focusZoneAncestor])
+			{
+				nextViewToFocus = nil;
+				break;
+			}
```

Comment:
Shouldn't this similar logic exist for shift-tab (the block below)? Presuming we'll crash/hang the same way there.

Reply:
Good point, I didn't check the shift-tab scenario. Will report back...

Reply:
Applied the same logic to `FocusZoneActionShiftTab` (duping the same code), there might be a way to make it cleaner though... And thanks for catching this early :)